### PR TITLE
tidy: drop `submission_email` column from `teams` table and adjust seed-database/write script

### PR DESCRIPTION
**Pull request by RODO94**
# What does this PR do?

To finish the work on https://trello.com/c/dVsTQpZZ/2984-allow-editors-to-see-update-their-teams-submission-email-in-team-settings-form

I have dropped the redundant column and adjusted the scripts which create the local DBs.

Ran regression testing and all passed

### CODE REVIEW

**Comment by RODO94** [comment]
Running Regression Testing on this PR since there's the potential for breaking changes.

File: hasura.planx.uk/migrations/1725611185485_alter_table_public_teams_drop_column_submission_email/down.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+comment on column "public"."teams"."submission_email" is E'Teams are core way we organise `flows` and configure settings like theme colour, logo, and contact info; name usually reflects a council but not exclusively';
+alter table "public"."teams" alter column "submission_email" drop not null;
+alter table "public"."teams" add column "submission_email" text;
```
Comment (jessicamcinchak) [inline hasura.planx.uk/migrations/1725611185485_alter_table_public_teams_drop_column_submission_email/down.sql:3]: I don't think this is a valid `down` migration - you can't `comment on` or `alter` a column before it exists/has been added.

I think this should be: 
```sql
alter table "public"."teams" add column "submission_email" text;
comment on column "public"."teams"."submission_email" is E'Teams are core way we organise `flows` and configure settings like theme colour, logo, and contact info; name usually reflects a council but not exclusively';
```

**Comment by RODO94** [comment]
[Allow editors to see/update their team's submission email in "Team settings" form](https://trello.com/c/dVsTQpZZ/2984-allow-editors-to-see-update-their-teams-submission-email-in-team-settings-form)

File: hasura.planx.uk/metadata/tables.yaml [hunk 2]
```diff
@@ -2130,35 +2128,8 @@
           - notify_personalisation
           - settings
           - slug
-          - submission_email
         filter: {}
         check: null
-- table:
-    name: teams_summary
-    schema: public
```
Comment (DafyddLlyr) [inline hasura.planx.uk/metadata/tables.yaml:2138]: I might well be missing context here, but there's no reference to dropping this table in the PR description. Is this correct?

This table is used for the admin summary page which still requires this table.

**Review by DafyddLlyr — CHANGES_REQUESTED** [review]

File: hasura.planx.uk/metadata/tables.yaml (hunk 2 above)
Comment (RODO94) [reply hasura.planx.uk/metadata/tables.yaml:2138]: It should only be the `submission_email` col, I'll revert this change.

Comment (RODO94) [reply hasura.planx.uk/metadata/tables.yaml]: @jessicamcinchak @DafyddLlyr this got removed due to the script for creating it having a reference to `teams.submission_email` - I am making a new PR as it's easier for reverting the changes. Sorry for missing it and thanks for picking it up.

https://github.com/theopensystemslab/planx-new/pull/3640

**Comment by RODO94** [comment]
Closing this branch and making changes on a cleaner one here: https://github.com/theopensystemslab/planx-new/pull/3640

**Pull request closed without merging**